Backstories/Backstories.xml

```diff
@@ -1501,79 +1501,79 @@ He was eventually betrayed by his superiors when a target found out about the hi
+		<title>Студент-историк</title>
+		<titleShort>Студент</titleShort>
+		<desc>Ещё ребёнком NAME запоминал(а) множество художественных образов, в том числе в музыке и романах. Когда HE подрос(ла), то начал(а) изучать легенды HIS родного мира и, в частности, легенды о похожих на ящериц существах, на которых был основан весь местный фольклор.</desc>
 	</HistoryStudent52>
 
 	<Storyteller68>
-		<title>Storyteller</title>
-		<titleShort>Fabulist</titleShort>
-		<desc>NAME traveled his world recounting the legends that he had studied.
+		<title>Рассказчик историй</title>
+		<titleShort>Фольклорист</titleShort>
+		<desc>NAME путешествовал(а) по своему миру, рассказывая о изученных легендах.
 
-Obsessed with the legends of his people, NAME seeks someone who can help him grow scales on his skin, so he too can be as the ancients were.</desc>
+HECAP был(а) одержим(а) древними сказаниями своего народа и искал(а) того, кто поможет HIM отрастить чешую на коже, как у древних.</desc>
 	</Storyteller68>
 
 	<NobleWard25>
-		<title>Noble ward</title>
-		<titleShort>Ward</titleShort>
-		<desc>NAME was adopted by a prominent noble family after being left on their doorstep by HIS mother.
+		<title>Благородный подкидыш</title>
+		<titleShort>Подкидыш</titleShort>
+		<desc>NAME был(а) усыновлен(а) высокопоставленной семьёй, у двери которой HIS оставила мать.
 
-HECAP quickly learned secrets that passed between the nobles like cheap wine - of worlds beyond her own, languages, cultures and technologies both new and old.</desc>
+HECAP быстро выучил(а) секреты, передававшиеся людьми благородными словно дешёвые вина — о далёких мирах, языках, культурах и технологиях старых и новых.</desc>
 	</NobleWard25>
 
 	<InformationBroker88>
-		<title>Information broker</title>
-		<titleShort>Broker</titleShort>
-		<desc>NAME caught a ship to the next star system and began a life of travel and learning, quickly collecting a massive bank of knowledge from people of all shapes and sizes.
+		<title>Информационный посредник</title>
+		<titleShort>Брокер</titleShort>
+		<desc>NAME нанял(а) корабль, следовавший к другой солнечной системе, и начал(а) жизнь, полную путешествий и обучения, быстро собирая массивную базу знаний от людей любоых народов и должностей.
 
-HECAP learned to sell this knowledge for great profit, and became a powerful broker of information.</desc>
+HECAP узнал(а), как продавать это знание задорого и стал(а) влиятельным информационным посредником.</desc>
 	</InformationBroker88>
 
 	<FeralChild63>
-		<title>Feral child</title>
-		<titleShort>Feral</titleShort>
-		<desc>Abandoned on an animal planet as a small child with nought but a blanket with a name embroidered on it, Felix made himself one with the wilderness.
+		<title>Дикий ребёнок</title>
+		<titleShort>Маугли</titleShort>
+		<desc>NAME был(а) совсем маленьким ребёнком, когда HIS оставили на планете, полной диких зверей, оставив лишь одеяло с HIS вышитым именем - NAME. HECAP стал(а) единым целым с природой.
```

Comment:
Abandoned on an animal planet as a small child with nought but a blanket with a name embroidered on it, Felix made himself one with the wilderness.